.github/workflows/pull-request-label.yml

```diff
@@ -0,0 +1,166 @@
+name: Pull Request Maintenance
+on:
+  pull_request_target:
+    # make sure that when the PR changes, we also update
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+permissions:
+  pull-requests: write
+  issues: write
+
+jobs:
+  conventional-commit-labeler:
+    name: Label PR based on Conventional Commit Specification
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7
+        env:
+          TYPE_TO_LABEL: |
+            {
+              "feat":"kind/feature",
+              "fix":"kind/bug",
+              "chore":"kind/chore",
+              "ref":"kind/refactor",
+              "clean":"kind/cleanup",
+              "design":"kind/design",
+              "docs":"kind/documentation",
+              "test":"area/testing",
+              "perf":"area/performance"
```

Comment:
weird to have two `areas` in here.. like

`test(testing): testing the testing` would provide two `area/testing` labels.. wouldn't that break?

Reply:
Yeah, because we don't have kind/testing and kind/performance. Should I add those rather? It would make more sense I think.

Reply:
I created kind/testing and kind/performance.